src/components/Main.js

```diff
@@ -99,7 +98,11 @@ function injectCSS (url) {
   document.getElementsByTagName('head')[0].appendChild(link);
 }
 
-(function init () {
+(function init() {
+  var webFont = document.createElement('script');
+  webFont.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js';
+  var webFontLoader = document.createElement('script');
```

Comment:
where do you actually add `webFontLoader` to `document.head`? how is this script getting loaded?